bears/python/MypyBear.py

```diff
@@ -0,0 +1,134 @@
+from collections import namedtuple
+import textwrap
+import sys
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+
+class FlagInfo(namedtuple('FlagInfo', 'arg doc inverse')):
+    """
+    Information about a command-line flag
```

Comment:
importance/low: periods at the end of sentences (since they're there in other places)